src/main/java/core/basesyntax/service/FileReadService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReadService {
+    List<String> readDataFromFile(String fileName);
```

Comment:
```suggestion
    List<String> readData(String fileName);
```
"File" in the class name is enough.